.github/workflows/cmake.yml

```
@@ -109,8 +109,8 @@ jobs:
       - name: Show ccache stats before build and configure
         run: |
           # We only have 5 GiB of cache available *in total*. Beyond that, GitHub deletes caches.
-          # Currently, we use about 130 MB for the two toolchains (i686 & x86_64), and three ccache caches:
-          # One with ALL_DEBUG (i686) and two with NORMAL_DEBUG (i686 & x86_64).
+          # Currently, we use about 130 MB for the toolchains (x86_64), and 2 ccache caches:
+          # Two with NORMAL_DEBUG (x86_64).
```

This grammar is wrong

```suggestion
          # Currently, we use about 130 MB for the toolchain (x86_64), and 2 ccache caches:
          # One with ALL_DEBUG (x86_64), and one with NORMAL_DEBUG (x86_64).
```